crates/chia-fuzzing/src/make_tree.rs

```diff
@@ -0,0 +1,184 @@
+use arbitrary::{Arbitrary, Unstructured};
+use clvmr::{Allocator, NodePtr};
+
+enum Op {
+    Pair(bool),
+    SubTree,
+}
+
+#[derive(Arbitrary)]
+enum NodeType {
+    Pair,
+    Bytes,
+    U8,
+    U16,
+    U32,
+    Previous,
+}
+
+#[allow(dead_code)]
+pub fn make_tree(a: &mut Allocator, unstructured: &mut Unstructured<'_>) -> (NodePtr, u32) {
+    make_tree_limits(a, unstructured, 600_000, true)
+}
+
+/// returns an arbitrary CLVM tree structure and the number of (unique) nodes
+/// it's made up of. That's both pairs and atoms.
+pub fn make_tree_limits(
+    a: &mut Allocator,
+    unstructured: &mut Unstructured<'_>,
+    mut max_nodes: i64,
+    reuse_nodes: bool,
+) -> (NodePtr, u32) {
+    let mut previous_nodes = Vec::<NodePtr>::new();
+    let mut value_stack = Vec::<NodePtr>::new();
+    let mut op_stack = vec![Op::SubTree];
+    // the number of Op::SubTree items on the op_stack
+    let mut sub_trees: i64 = 1;
+    let mut counter = 0;
+
+    while let Some(op) = op_stack.pop() {
+        match op {
+            Op::Pair(swap) => {
+                let left = value_stack.pop().expect("internal error, empty stack");
```

Comment:
I know it's fuzzing code so we don't care about handling errors (ie panicking is fine), but to make the code simpler you can return an `anyhow::Result` anyway and use the `?` operator instead of using `expect` on `Result` types. At the call site this whole function can be `.expect`ed.

Reply:
@jack60612 I see you return `anyhow` from this function, but you didn't implement the suggestion of replacing `.expect()` with `?`. Was this deliberate?